app/views/observations/show/_observation_details.html.erb

```diff
@@ -17,22 +17,16 @@ caption ||= false
 
 <p class="obs-lat-lng" id="observation_lat_lng">
   <%=
-    hidden = "<i>(#{:show_observation_gps_hidden.t})</i>".html_safe
-    if obs.lat &&
-        (!obs.gps_hidden || obs.can_edit?)
+    if obs.reveal_location?
```

Comment:
Holy moly, this is a lot cleaner than what was here before!!

Reply:
I know, right?